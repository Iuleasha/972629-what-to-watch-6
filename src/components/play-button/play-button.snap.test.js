import {render} from '@testing-library/react';
import React from 'react';
import PlayButton from './play-button';

test(`Should PlayButton render correctly`, () => {
  const {container} = render(<PlayButton id={`1`}/>);
  expect(container).toMatchSnapshot();
});
