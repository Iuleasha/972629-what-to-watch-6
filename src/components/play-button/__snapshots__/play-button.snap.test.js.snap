// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should PlayButton render correctly 1`] = `
<div>
  <button
    class="btn movie-card__button"
    type="button"
  >
    <svg
      height="20"
      viewBox="0 0 19 20"
      width="19"
    >
      <use
        xlink:href="#play-s"
      />
    </svg>
    <span>
      Play
    </span>
  </button>
</div>
`;
